redis/asyncio/connection.py

```diff
@@ -56,15 +56,18 @@
 if HIREDIS_AVAILABLE:
     import hiredis
 
-NONBLOCKING_EXCEPTION_ERROR_NUMBERS = {
-    BlockingIOError: errno.EWOULDBLOCK,
-    ssl.SSLWantReadError: 2,
-    ssl.SSLWantWriteError: 2,
-    ssl.SSLError: 2,
-}
+if sys.version_info[:2] >= (3, 10):
```

Comment:
nullcontext is available as of 3.7, but the [async context manager](https://docs.python.org/3/glossary.html#term-asynchronous-context-manager) is available since 3.10

Reply:
This has been removed.